ci/verify/release-pipeline.yaml

```diff
@@ -56,7 +56,14 @@ spec:
         password: ((pipeline.ImageRegistryPassword))
         repository: ((metadata-controller.ImageRepositoryURI))
 
-    - name: cloudhsm-image
+    - name: cloudhsm-client-src
+      type: github
+      source:
+        <<: *github_source
```

Comment:
if we're going to use this we should probably override `repository` too

Reply:
cool done